kinto/__init__.py

```diff
@@ -53,6 +53,17 @@ def main(global_config, config=None, **settings):
     # Retro-compatibility with first Kinto clients.
     config.registry.public_settings.add('cliquet.batch_max_requests')
 
+    # Expose capability
+    schema_enabled = asbool(
+        settings.get('experimental_collection_schema_validation')
```

Comment:
nit: since default settings are loaded, you can do  `settings['experimental...]`